server/website/website/settings/common.py

```diff
@@ -6,6 +6,7 @@
 import os
 import sys
 from os.path import abspath, dirname, exists, join
+import djcelery
```

Comment:
I think you can remove this import now that you set the djcelery test runner elsewhere. Actually it looks like you don't need to make any changes to this file for this PR.